HHS/Head-Start-TTADP
# [Prod] Fix bug with FEI goals, fix bug with Slack urls, remove resource topic table, use report end date for goal status change date

## Description of change
- Fixed a bug where start date ranges weren't correctly parsed when visiting the site via pasted link.
- Make sure that on multi-recipient reports, the FEI goal's conditional fields are not passed back and forth, regardless of when the report became a multi-recipient report
- Remove the resource topic table from the resource dashboard
- Use the report end date as the value for the firstInProgressAt and lastInProgressAt columns under certain limited circumstances.

## How to test
- Paste a filtered link to the activity reports landing page into slack (I messaged myself) and then click on it. You should see the proper values reflected in the filter pills.
- Create a report but add a second recipient after filling out the FEI goals. The report should save as normal.
- Make sure the topics widget is not on the resource dashboard.
- Create a report, select an end date different than today, create a new goal, and approve the report. The value for "firstInProgressAt" and "lastInProgressAt" should be updated to use the end date for the report.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1693
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1219
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1713
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1683


## Checklists

### Every PR


- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [n/a] UI review complete

### Production Deploy


### After merge/deploy
